app/controllers/hosts_controller.rb

```diff
@@ -15,7 +15,7 @@ def show
     respond_to do |format|
       format.html { correct_host }
       format.json { 
-        if @host.strangers && @host.available_places > 0 
```

Comment:
just to explain the change:
when you click סגור סלום, basicall we don't allow strangers anymore, but there are still places.
by removing the strangers condition, even when you close הרשמה, you still see the poeple waiting. only when you reach limit of approved people, is this hidden